cognite/client/_api/synthetic_time_series.py

```diff
@@ -10,17 +10,16 @@
 from cognite.client.utils._experimental_warning import experimental_api
 
 
-@experimental_api(api_name="Synthetic Timeseries")
 class SyntheticDatapointsAPI(APIClient):
-    _SYNTHETIC_RESOURCE_PATH = "/timeseries/synthetic"
+    _RESOURCE_PATH = "/timeseries/synthetic"
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._DPS_LIMIT = 10000
 
     def retrieve(
         self,
-        expression: Union[str, "sympy.Expr"],
+        expressions: List[Union[str, "sympy.Expr"]],
```

Comment:
I'd like this to accept a single expression/string as well for convenience/backward compatibility. @hhenriques thoughts?

Reply:
agreed

Reply:
@sanderland, done